web/impact/impact/v1/views/user_detail_view.py

```diff
@@ -58,8 +58,7 @@ def patch(self, request, pk):
         valid_keys = set(data.keys()).intersection(UserHelper.INPUT_KEYS)
         valid_data = {}
         for key in valid_keys:
-            field = helper.translate_key(key)
-            valid_data[field] = helper.validate(field, data[key])
+            valid_data[key] = helper.validate(key, data[key])
```

Comment:
why was the translate key step eliminated? is it because we are just manually changing the references?

Reply:
No longer needed because we don't have short_name/last_name any more which is all this mechanism was getting used for.